app/persistence/fabric/CRUDService.ts

```diff
@@ -475,6 +475,30 @@ export class CRUDService {
 		}
 	}
 	// Orderer BE-303
+
+	/**
+	 *
+	 * Returns the block by block number.
+	 *
+	 * @param {*} channel_genesis_hash
+	 * @param {*} blockNo
+	 * @returns
+	 * @memberof CRUDService
+	 */
+	async getBlockByBlocknum(network_name, channel_genesis_hash, blockNo) {
```

Comment:
[nitpick]: types can be provided in the function args.

Reply:
@adityajoshi12  Followed the same coding styles which existed and couldn't find any function args with types defined. Do I need to add types for this function?

Reply:
Since the project is in typescript, let have the types going forward.